src/app/notification/toast-notification-list.component.html

```diff
@@ -0,0 +1,16 @@
+<div class="toast-notifications-list-pf" *ngIf="notifications && notifications.length > 0">
+  <div *ngFor="let notification of notifications">
+    <alm-toast-notification
```

Comment:
you should start using 'pf-' for patternfly if these are going to go there instead of 'alm-'